eos-tech-support/eos-download-image

```diff
@@ -25,7 +25,11 @@ def log(*args):
 
 
 def check_call(*args):
-    log('$ ' + ' '.join(map(shlex.quote, args)))
+    if shlex.quote:
+        qargs = map(shlex.quote, args)
+    else:
+        qargs = args
```

Comment:
Actually, it appears that `shlex.quote` bombs if the args aren't strings, so maybe it doesn't matter:
```
>>> shlex.quote(['foo', 1, 2])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/lib/python3.4/shlex.py", line 285, in quote
    if _find_unsafe(s) is None:
TypeError: expected string or buffer
```